web/src/app/lists/SimpleListPage.tsx

```diff
@@ -1,33 +1,27 @@
-import React, { useState, ReactElement } from 'react'
+import React, { ComponentType, useState } from 'react'
 import QueryList, { QueryListProps } from './QueryList'
 import CreateFAB from './CreateFAB'
 
 interface SimpleListPageProps extends QueryListProps {
-  createForm: ReactElement
+  createDialogComponent: ComponentType<{ onClose: () => void }>
   createLabel: string
 }
 
 export default function SimpleListPage(
   props: SimpleListPageProps,
 ): JSX.Element {
-  const { createForm, createLabel, ...rest } = props
+  const { createDialogComponent, createLabel, ...rest } = props
   const [create, setCreate] = useState(false)
+  const DialogComponent = createDialogComponent
```

Comment:
this syntax should be legal:

```suggestion
  const { createDialogComponent: DialogComponent, createLabel, ...rest } = props
  const [create, setCreate] = useState(false)
```